.github/workflows/ci.yml

```diff
@@ -46,7 +46,7 @@ jobs:
         run: yarn test
 
   test-esm:
-    name: Testing with Node ESM
+    name: esm
```

Comment:
@dotansimha Isn't Testing with Node ESM nicer @dotansimha ? 😅  I like the existing names to be honest

Reply:
They are too long, that's my main issue with it... `test / esm` is kinda clear no?